src/command_modules/azure-cli-dls/azure/cli/command_modules/dls/_params.py

```diff
@@ -64,6 +64,7 @@ def load_arguments(self, _):
     #####
     with self.argument_context('dls account network-rule') as c:
         c.argument('account_name', datalake_store_name_type, options_list=['--account-name'], id_part=None)
+        c.argument('resource_group_name', resource_group_name_type, id_part='name')
```

Comment:
id_part for resource group name should definitely not be name. It should only be `resource_group` (at least in this context)

Reply:
Thanks, changed.
```
(venv) E:\Users\lewu\workspace\lewu-msft\azure-cli>az dls account network-rule delete -h

Command
    az dls account network-rule delete : Deletes a virtual network rule in a Data Lake Store
    account.

Arguments
    --account-name [Required] : Name of the Data Lake Store account.

Resource Id Arguments
    --ids                     : One or more resource IDs (space-delimited). If provided, no other
                                'Resource Id' arguments should be specified.
    --name -n                 : The virtual network rule name.
    --resource-group -g       : Name of resource group. You can configure the default group using
                                `az configure --defaults group=<name>`.

Global Arguments
    --debug                   : Increase logging verbosity to show all debug logs.
    --help -h                 : Show this help message and exit.
    --output -o               : Output format.  Allowed values: json, jsonc, table, tsv, yaml.
                                Default: json.
    --query                   : JMESPath query string. See http://jmespath.org/ for more information
                                and examples.
    --subscription            : Name or ID of subscription. You can configure the default
                                subscription using `az account set -s NAME_OR_ID`.
    --verbose                 : Increase logging verbosity. Use --debug for full debug logs.


(venv) E:\Users\lewu\workspace\lewu-msft\azure-cli>az dls account network-rule show -h

Command
    az dls account network-rule show : Get the details of a virtual network rule in a Data Lake
    Store account.

Arguments
    --account-name [Required] : Name of the Data Lake Store account.

Resource Id Arguments
    --ids                     : One or more resource IDs (space-delimited). If provided, no other
                                'Resource Id' arguments should be specified.
    --name -n                 : The virtual network rule name.
    --resource-group -g       : Name of resource group. You can configure the default group using
                                `az configure --defaults group=<name>`.

Global Arguments
    --debug                   : Increase logging verbosity to show all debug logs.
    --help -h                 : Show this help message and exit.
    --output -o               : Output format.  Allowed values: json, jsonc, table, tsv, yaml.
                                Default: json.
    --query                   : JMESPath query string. See http://jmespath.org/ for more information
                                and examples.
    --subscription            : Name or ID of subscription. You can configure the default
                                subscription using `az account set -s NAME_OR_ID`.
    --verbose                 : Increase logging verbosity. Use --debug for full debug logs.


(venv) E:\Users\lewu\workspace\lewu-msft\azure-cli>az dls account network-rule update -h

Command
    az dls account network-rule update : Updates a virtual network rule in a Data Lake Store
    account.

Arguments
    --account-name [Required] : Name of the Data Lake Store account.
    --subnet       [Required] : Name or ID of the subnet that allows access to DLS. If subnet name
                                is provided, --name must be provided.
    --vnet-name               : The virtual network rule name.

Generic Update Arguments
    --add                     : Add an object to a list of objects by specifying a path and key
                                value pairs.  Example: --add property.listProperty <key=value,
                                string or JSON string>.
    --force-string            : When using 'set' or 'add', preserve string literals instead of
                                attempting to convert to JSON.
    --remove                  : Remove a property or an element from a list.  Example: --remove
                                property.list <indexToRemove> OR --remove propertyToRemove.
    --set                     : Update an object by specifying a property path and value to set.
                                Example: --set property1.property2=<value>.

Resource Id Arguments
    --ids                     : One or more resource IDs (space-delimited). If provided, no other
                                'Resource Id' arguments should be specified.
    --name -n                 : The virtual network rule name.
    --resource-group -g       : Name of resource group. You can configure the default group using
                                `az configure --defaults group=<name>`.

Global Arguments
    --debug                   : Increase logging verbosity to show all debug logs.
    --help -h                 : Show this help message and exit.
    --output -o               : Output format.  Allowed values: json, jsonc, table, tsv, yaml.
                                Default: json.
    --query                   : JMESPath query string. See http://jmespath.org/ for more information
                                and examples.
    --subscription            : Name or ID of subscription. You can configure the default
                                subscription using `az account set -s NAME_OR_ID`.
    --verbose                 : Increase logging verbosity. Use --debug for full debug logs.


(venv) E:\Users\lewu\workspace\lewu-msft\azure-cli>az dls account network-rule create -h

Command
    az dls account network-rule create : Creates a virtual network rule in a Data Lake Store
    account.

Arguments
    --account-name [Required] : Name of the Data Lake Store account.
    --name -n      [Required] : The virtual network rule name.
    --subnet       [Required] : The subnet name or id for the virtual network rule.
    --resource-group -g       : Name of resource group. You can configure the default group using
                                `az configure --defaults group=<name>`.
    --vnet-name               : The name of the virtual network rule.

Global Arguments
    --debug                   : Increase logging verbosity to show all debug logs.
    --help -h                 : Show this help message and exit.
    --output -o               : Output format.  Allowed values: json, jsonc, table, tsv, yaml.
                                Default: json.
    --query                   : JMESPath query string. See http://jmespath.org/ for more information
                                and examples.
    --subscription            : Name or ID of subscription. You can configure the default
                                subscription using `az account set -s NAME_OR_ID`.
    --verbose                 : Increase logging verbosity. Use --debug for full debug logs.


(venv) E:\Users\lewu\workspace\lewu-msft\azure-cli>az dls account network-rule list -h

Command
    az dls account network-rule list : Lists virtual network rules in a Data Lake Store account.

Arguments
    --account-name [Required] : Name of the Data Lake Store account.
    --resource-group -g       : Name of resource group. You can configure the default group using
                                `az configure --defaults group=<name>`.

Global Arguments
    --debug                   : Increase logging verbosity to show all debug logs.
    --help -h                 : Show this help message and exit.
    --output -o               : Output format.  Allowed values: json, jsonc, table, tsv, yaml.
                                Default: json.
    --query                   : JMESPath query string. See http://jmespath.org/ for more information
                                and examples.
    --subscription            : Name or ID of subscription. You can configure the default
                                subscription using `az account set -s NAME_OR_ID`.
    --verbose                 : Increase logging verbosity. Use --debug for full debug logs.
```